client/transaction-pool/src/testing/pool.rs

```diff
@@ -263,14 +262,10 @@ fn should_not_retain_invalid_hashes_from_retracted() {
 
 	let event = block_event_with_retracted(1, vec![retracted_hash]);
 
-	notifier.clear();
 	block_on(pool.maintain(event));
 	// maintenance is in background
-	block_on(notifier.next());
-
-	let event = block_event_with_retracted(1, vec![retracted_hash]);
```

Comment:
We lost one test case it seems?

Reply:
it was duplicated code that didn't add anything